library/src/main/kotlin/com/chuckerteam/chucker/internal/data/entity/HttpTransaction.kt

```diff
@@ -55,6 +55,7 @@ internal class HttpTransaction(
     @ColumnInfo(name = "isResponseBodyEncoded") var isResponseBodyEncoded: Boolean = false,
     @ColumnInfo(name = "responseImageData") var responseImageData: ByteArray?,
     @ColumnInfo(name = "graphQlOperationName") var graphQlOperationName: String?,
+    @ColumnInfo(name = "isGraphQLRequest") var isGraphQLRequest: Boolean = false,
```

Comment:
```suggestion
    @ColumnInfo(name = "graphQLDetected") var graphQLDetected: Boolean = false,
```

So we keep the `graphQL*` prefix for GraphQL related properties